scripts/issue/729/deprecate_polygon.py

```diff
@@ -0,0 +1,30 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+TECH_OPS = registry.poly.badger_wallets.techops_multisig
+
+REGISTRY = registry.poly.registry_v2
+
+WBTC_IBBTC_SLP = registry.poly.sett_vaults.bslpibBTCWbtc
+WBTC_USDC_QLP = registry.poly.sett_vaults.bqlpUsdcWbtc
+AWBTC_REN_CRV = registry.poly.sett_vaults.bcrvRenBTC
+
+def deprecate_polygon():
```

Comment:
would opt for using default `main` here if there is no good reason not to. this saves having to specify the func name when calling the script:
```
brownie run issue/729/deprecate_polygon deprecate_polygon
```
vs
```
brownie run issue/729/deprecate_polygon
```